fe/src/app/contact/page.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Card,
+  Input,
+  Button,
+  Typography,
+} from "@/lib/materialTailwindExports";
+
+const ContactPage = () => {
+  return (
+    <div className="bg-gray-800 text-white flex h-screen p-8 md:p-16">
+      <main className="flex-1 py-12 text-center">
+        <h1 className="text-7xl md:text-7xl font-bold mb-12">Contact Us</h1>
+        <div className="max-w-2xl mx-auto text-xl md:text-2xl leading-relaxed">
+          <p className="mb-12">
+            We{'\''}re here to assist you! Feel free to reach out if you have any questions, suggestions, or just want to say hello.
+          </p>
+          <p className="mb-12">
+            Connect with us through the following channels:
+          </p>
+          <ul className="list-disc list-inside mb-8">
+            <li>Email: <a href="mailto:info@example.com" className="text-blue-500 hover:underline">CarFans@example.com</a></li>
```

Comment:
For me, those bullet points look ugly, we can remove them with this property `list-none` for ul tag. Also I would not mind if we aligned them in left not center like now.

![image](https://github.com/MatasAdmoneo/CarFans/assets/80633152/627d522e-3a51-4831-a2d7-9597950f3908)

Reply:
Also not sure if we need link for email.